python/pyarrow/tests/test_array.py

```diff
@@ -1766,47 +1763,6 @@ def test_list_array_flatten():
     assert arr2.flatten().flatten().equals(arr0)
```

Comment:
Now `.values`is implemented separately from `.flatten()`, can you add asserts for this property as well?

Reply:
Will do.